start_servers.sh

```
@@ -0,0 +1,181 @@
+# Function to get process info for a port
+get_port_process() {
+    local port=$1
+    lsof -i :$port | grep LISTEN | awk '{print $2, $1}' | head -1
+}
+
+# Function to kill process on port
+kill_port_process() {
+    local port=$1
+    echo -e "${YELLOW}🔍 Finding process on port $port...${NC}"
+    
+    local process_info=$(get_port_process $port)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)